src/js/components/query_mediator.js

```diff
@@ -213,7 +213,7 @@ define(['underscore',
           return;
         }
 
-        if (apiQuery.get("q")[0].indexOf("simbid:") > -1) {
```

Comment:
some simbid searches seemed to have simbid0 instead of simbid:, not sure if I missed something, but I removed the colon to catch those cases